mon-pix/tests/unit/components/authentication/oidc-reconciliation_test.js

```diff
@@ -131,4 +133,48 @@ describe('Unit | Component | authentication | oidc-reconciliation', function ()
       });
     });
   });
+
+  context('when authentication key has expired', function () {
+    it('should display error', async function () {
+      // given
+      const component = createGlimmerComponent('component:authentication/oidc-reconciliation');
+      const authenticateStub = sinon.stub().rejects({ errors: [{ status: '401' }] });
+      class SessionStub extends Service {
+        authenticate = authenticateStub;
+      }
+      this.owner.register('service:session', SessionStub);
+      component.args.identityProviderSlug = 'super-idp';
+      component.args.authenticationKey = 'super-key';
+      component.isTermsOfServiceValidated = true;
+
+      // when
+      await component.reconcile();
+
+      // then
+      expect(component.reconcileErrorMessage).to.equal(
+        this.intl.t('pages.login-or-register-oidc.error.expired-authentication-key')
+      );
+    });
+  });
+
+  context('when an error happens', function () {
+    it('should display error', async function () {
```

Comment:
Cool que ce soit testé ^^ !

Reply:
On pourrait mettre 'should display generic error message` (si generique existe en anglais)